src/frontend/src/lib/utils/network.utils.ts

```diff
@@ -33,6 +33,15 @@ export const isNetworkIdBTCTestnet = (networkId: NetworkId | undefined): boolean
 export const isNetworkIdBTCRegtest = (networkId: NetworkId | undefined): boolean =>
 	BTC_REGTEST_NETWORK_ID === networkId;
 
+const mapper: Record<symbol, BitcoinNetwork> = {
+	[BTC_MAINNET_NETWORK_ID]: 'mainnet',
+	[BTC_TESTNET_NETWORK_ID]: 'testnet',
+	[BTC_REGTEST_NETWORK_ID]: 'regtest'
+};
+
+export const mapNetworkIdToBitcoinNetwork = (networkId: NetworkId): BitcoinNetwork | undefined =>
```

Comment:
can it happen that it is `undefined`? I mean, would it give some error if it's not the existing Keys? I sincerly don't know, but if that's the case, maybe create a test for this specific case?

Reply:
Done!